File: projektvorlage/go/features/vorlage/handlers.go
// Verhalten dieses Features. I/O ist hier erlaubt.
package template

import "PROJEKTNAME/atoms/vorlage/example"

func Handle(value int) (Example, error) {
	v, err := example.Run(value)
	if err != nil {
		return Example{}, err
	}
	return Example{Value: v}, nil
}

File: projektvorlage/rust/src/features/vorlage/handlers.rs
//! Verhalten dieses Features. I/O ist hier erlaubt.

use crate::atoms::vorlage::example::run as identity;

use super::models::Example;

pub fn handle(value: i64) -> Result<Example, String> {
    Ok(Example { value: identity(value)? })
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn baut_das_modell() {
        assert_eq!(handle(7), Ok(Example { value: 7 }));
    }
}
